README.md

```diff
@@ -111,12 +111,26 @@ If you want to remove this stack from AWS and delete everything, run `npm run re
 
 The [Serverless docs on workflow are useful](https://serverless.com/framework/docs/providers/aws/guide/workflow/).
 
+### Custom stable domain name for local development
+
+By default, no custom domain name is created. You can use the semi-random domain name serverless offers on deployment and with `serverless info`.
+
+If you want to create a domain name for local development (e.g. `watchdog-proxy-lmorchard.dev.mozaws.net`):
+
+1. Edit your `serverless.local.yml` to contain an enabled `customDomain` section with appropriate details
+1. Run `npx serverless create_domain` - this only needs to be done once, to create the new custom domain name in Route53 and an accompanying CloudFront distribution
+1. Run `npm run deploy:dev` to update your stack
```

Comment:
IIRC previously the README had an example of passing the stage name with `--stage`.  If that argument is missing and the `STAGE` env var does not exist, then `npm run deploy:dev` would produce the following error:

`An error occurred: pathmapping - Invalid stage identifier specified (Service: AmazonApiGateway; Status Code: 400; Error Code: BadRequestException; Request ID: ...).`

Perhaps we should mention that a stage name is required for this command?

(https://github.com/serverless/serverless/issues/4029 might be relevant.)

Reply:
Hmm, I think I assumed here that you'd have stage in a `serverless.local.yml` - that was meant to take the place of the `--stage` CLI option. The `STAGE` env var is mainly just for npm scripts where it's easier to pass vars than CLI options, and where `serverless.local.yml` isn't created. 

Are you getting this error with a `serverless.local.yml` created?

Reply:
Oh right!  I remember that change now.  Yeah, I overwritten my existing local config with the dist file for the `customDomain` stuff, but then forgot to change the `stage` value.  More specifically `stage` was not matching customDomain's `stage`.

My bad.

Reply:
Yeah, this stuff might be getting more complicated than it needs to be :/